ios/Classes/SwiftOneginiPlugin.swift

```diff
@@ -113,31 +112,27 @@ public class SwiftOneginiPlugin: NSObject, FlutterPlugin, UserClientApi {
     }
 
     func pinDenyAuthenticationRequest(completion: @escaping (Result<Void, Error>) -> Void) {
-        OneginiModuleSwift.sharedInstance.cancelPinAuth()
-        // FIXME: in the above function the completion is actually not yet used as that would create way to big of a refactor, so let's do it later in FP-49
-        completion(.success)
+        OneginiModuleSwift.sharedInstance.pinDenyAuthenticationRequest() { result in
+            completion(result.mapError{$0})

```

Comment:
`completion(result.mapError { $0 })`

Reply:
Wait, isn't result == #0 ? 🤔

Reply:
yes it is, we had a conversation about this a while ago. The compiler somehow doesn't understand that these types are the same and we have to help it out a bit here. 
If you would pass in result into the completion directly it would complain that FlutterError is not convertable to Error while clearly it is, because FlutterError conforms to the Error protocol.

Reply:
hmm even though the syntax is horrible. you could have 
```
OneginiModuleSwift.sharedInstance.pinDenyAuthenticationRequest() { $0.mapError { $0 } )
```
or
```
OneginiModuleSwift.sharedInstance.pinDenyAuthenticationRequest() { result in 
result.mapError { result }
)
```
or even refactor `mapError` method to static so you could have sth like 
```
OneginiModuleSwift.sharedInstance.pinDenyAuthenticationRequest() { FlutterError.mapError { $0 })
```
What do you think?